module-3/backend/week_two.md

```diff
@@ -4,9 +4,28 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-* What's OAuth?
-* What are some advantages/disadvantages of implementing OAuth?
-* How many exhanges of information need to take place before a user is authenticated with OAuth?
-* Why do we want to create services?
-* Why is it good practice to create PORO's with the data received from an API?
-* What do we use VCR for? Why is a good idea to use this tool?
+1. What's OAuth?
+2. What are some advantages/disadvantages of implementing OAuth?
+3. How many exhanges of information need to take place before a user is authenticated with OAuth?
+4. Why do we want to create services?
+5. Why is it good practice to create PORO's with the data received from an API?
+6. What do we use VCR for? Why is it a good idea to use this tool?  
+
+#### Review  
+
+7. What does HTTP stand for?  
+8. How do you create the following table in SQL? In Active Record?   
+   (Users table with columns first_name, last_name, email, and age)   
+9. Writing Files: given a text file located at "~/Documents/pizza.txt", write code to read the file from the filesystem, then    write a new file at "~/Documents/line_count.txt" containing the number of lines in the original file.  
```

Comment:
Would you rather this be another enumerable question? Perhaps this group_by question?
Given the the array (alphabetical = ["aardvark", "art", "airplane", "boy", "burp", "boot", "green", "goop", "super"]), create a Hash where the keys are the first letter of words, and the values are the list of words that share that first letter.